test/e2e/projects_test.go

```diff
@@ -38,7 +38,7 @@ const (
 	nonExistingProjectName = "nonExistingProjectNameCAPX"
 )
 
-var _ = Describe("Nutanix projects [PR-Blocking]", func() {
+var _ = Describe("Nutanix projects", func() {
```

Comment:
IIRC, we didn't want projects to be PR blocking so that makes sense.

Reply:
correct, its not per this code as well